pkg/trtl/server_test.go

```diff
@@ -196,12 +195,10 @@ func (s *trtlTestSuite) setupConfig() (err error) {
 			ReindexOnBoot: false,
 		},
 		Replica: config.ReplicaConfig{
-			Enabled:        true,
-			PID:            metaPID,
-			Region:         metaRegion,
-			Name:           metaOwner,
-			GossipInterval: 1 * time.Second,
-			GossipSigma:    100 * time.Millisecond,
```

Comment:
Removed because we just want to use the default values here?

Reply:
Yes, we're no longer testing the replica in `trtl` - the tests are in `replica`, so this configuration was unnecessary. Removed it for clarity.